.golangci.yml

```diff
@@ -17,12 +17,14 @@ linters:
   - bidichk
   - bodyclose
   - containedctx
+  - decorder
   - depguard
   - dogsled
   - dupword
   - durationcheck
   - errcheck
   - errchkjson
+  - execinquery
```

Comment:
https://github.com/1uf3/execinquery
"execinquery - a simple query string checker in Query function" 

I don't think we need this since we don't use sql. @sbueringer WDYT?

Reply:
+1  -  I think we shouldn't include this

Reply:
Removed.